packages/shortest/src/core/framework-detector/index.ts

```diff
@@ -85,3 +103,22 @@ export const detectFramework = async (options: { force?: boolean } = {}) => {
     throw new ShortestError("Failed to save project information");
   }
 };
+
+const detectNextJsDirPathFromConfig = async (): Promise<string | undefined> => {
```

Comment:
Best-effort in detecting the folder of a Next.js app. It now uses `next.config.*` files for reference.